### ISSUE TITLE
Posting to MS Teams for "releases" 

### ISSUE THREAD

**Issue by antondimitrov** [integrations/microsoft-teams#39]
We often have to communicate the availability of a new release/hotfix to a group of stakeholders.

It would be very handy to offer a `subscription`  to a `release` that allows you to configure messages to a Teams channel where users get notified about a new release of `Product X` being available as well as details of the description of that release as they are in GithHub

**Comment by ashokirla** [comment]
@antondimitrov Yes, notifications for releases is in our backlog. We will work be working on it in next quarter.

**Comment by joelhoisko** [comment]
Hi! Any news on this feature so far? We've also just established pipelines for using GitHub Releases with descriptions and would love to automatically notify about new releases to a Teams channel etc.

**Comment by antondimitrov** [comment]
Hi @ashishonce  👋  
Do you have nay update on this feature request and can we expect it to be released this quarter as per your remark above ?

**Comment by ashishonce** [comment]
@antondimitrov i think you wanted to mention @ashokirla

**Comment by antondimitrov** [comment]
> @antondimitrov i think you wanted to mention @ashokirla

Yes I did! Thanks for pointing it out :)

**Comment by lucahost** [comment]
hey, I'm also interested in this feature, would you mind sharing some details of a planned roll out?

**Comment by antondimitrov** [comment]
@ashokirla,  Is this planned anywhere in the quarter ?  It seems that more people are seeing the need for this.

**Comment by cquanu** [comment]
Me too

**Comment by romainb** [comment]
Hi, this feature would be really helpful for me as well.

**Comment by cquanu** [comment]
Any idea when this will be implemented?

**Comment by davecozz** [comment]
I tried to setup a "Release Notifications" channel in Teams this morning and was very surprised to see that Release events were not showing up. Is there any ETA yet as to when this missing feature will be available?

**Comment by JeffLoo-ong** [comment]
@ashokirla our teams would greatly appreciate this feature! We saw that there are still a number of things unavailable but hopefully this can get some love soon... :)

**Comment by ickmoza** [comment]
> @antondimitrov Yes, notifications for releases is in our backlog. We will work be working on it in next quarter.

Did this ever happen? This comment was from 2020

**Comment by sc-addypathania** [comment]
You can create a webhook for your teams channel and then configure that webhook on the desired repository in github.
You can follow [this blog](https://www.indiumsoftware.com/blog/github-event-request-notification-to-teams-channel/) for help.
